spec/MediatorManager+services.yaml

```diff
@@ -1,293 +1,296 @@
+      uuid: mm-1-0-0-tcp-c-aa-2-0-1-000
     operation-clients:
       own-oam:
         basic:
           - operation-name: /v1/approve-oam-request
-            uuid: xx-1-0-0-op-c-bs-aa-2-0-1-000
+            uuid: mm-1-0-0-op-c-bs-aa-2-0-1-000
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
         individual:
-          ## TODO: Add Clients specific to this application
 
   - http-client:
       application-name: ApplicationLayerTopology
       release-number: 2.0.1
-      uuid: xx-1-0-0-http-c-alt-2-0-1-000
+      uuid: mm-1-0-0-http-c-alt-2-0-1-000
     tcp-client:
       remote-protocol: http
       ip-address: 1.1.3.13
       tcp-port: 3013
-      uuid: xx-1-0-0-tcp-c-alt-2-0-1-000
+      uuid: mm-1-0-0-tcp-c-alt-2-0-1-000
     operation-clients:
       own-oam:
         basic:
           - operation-name: /v1/update-all-ltps-and-fcs
-            uuid: xx-1-0-0-op-c-bm-alt-2-0-1-000
+            uuid: mm-1-0-0-op-c-bm-alt-2-0-1-000
           - operation-name: /v1/update-ltp
-            uuid: xx-1-0-0-op-c-bm-alt-2-0-1-001
+            uuid: mm-1-0-0-op-c-bm-alt-2-0-1-001
           - operation-name: /v1/delete-ltp-and-dependents
-            uuid: xx-1-0-0-op-c-bm-alt-2-0-1-002
+            uuid: mm-1-0-0-op-c-bm-alt-2-0-1-002
           - operation-name: /v1/update-fc
-            uuid: xx-1-0-0-op-c-bm-alt-2-0-1-003
+            uuid: mm-1-0-0-op-c-bm-alt-2-0-1-003
           - operation-name: /v1/update-fc-port
-            uuid: xx-1-0-0-op-c-bm-alt-2-0-1-004
+            uuid: mm-1-0-0-op-c-bm-alt-2-0-1-004
           - operation-name: /v1/delete-fc-port
-            uuid: xx-1-0-0-op-c-bm-alt-2-0-1-005
+            uuid: mm-1-0-0-op-c-bm-alt-2-0-1-005
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
         individual:
-          ## TODO: Add Clients specific to this application
 
   - http-client:
       application-name: OperationKeyManagement
       release-number: 2.0.1
-      uuid: xx-1-0-0-http-c-okm-2-0-1-000
+      uuid: mm-1-0-0-http-c-okm-2-0-1-000
     tcp-client:
       remote-protocol: http
       ip-address: 1.1.3.14
       tcp-port: 3014
-      uuid: xx-1-0-0-tcp-c-okm-2-0-1-000
+      uuid: mm-1-0-0-tcp-c-okm-2-0-1-000
     operation-clients:
       own-oam:
         basic:
         individual:
       service:
         basic:
-          ## TODO: Add Clients specific to this application, but addressing all other applications
         individual:
-          ## TODO: Add Clients specific to this application
+
 
   - http-client:
-      application-name: ## TODO: Add Name of application to be addressed
-      release-number: ## TODO: Add Release number of application to be addressed
-      uuid: ## TODO: Add UUID of the HttpClient
+      application-name: xMediatorInstanceManager #ask Thorsten they are indv
```

Comment:
before merging we should clarify this point with Thorsten and add the client stacks according to his feedback